openlibrary/templates/books/custom_carousel.html

```diff
@@ -63,7 +63,7 @@ <h2 class="home-h2"><a href="$url">$title</a></h2>
         $if load_more:
           $ load_more_config = {
           $  "url": load_more["url"].replace("&amp;", "&"),
-          $  "pageMode": load_more.get("mode", "offset"),
+          $  "pageMode": load_more.get("mode", "page"),
```

Comment:
. I'm not sure if `offset` is used on another page with a carousel.. can you confirm confidently (I can't :-)) ? If not, parameter to the macro would be safer.

Reply:
I think if we can keep offset.. even better.

Reply:
Oh nice!